README.md

```diff
@@ -11,10 +11,16 @@ Checkout the section, *Why Jekyll?*, below for more.
 ### Development
 ```bash
 # Clone the repository
-$ git clone git@github.com:KappaEtaKappa/delta.khk.org.jekyll.git
+$ git clone git@github.com:KappaEtaKappa/delta.khk.org.git
+$ cd delta.khk.org
+
+# Download and build all project dependencies into a local folder
+$ bundle install --path vendor/bundle
 
 # Run the Jekyll development server (http://localhost:1924)
-$ cd delta.khk.org.jekyll
+$ bundle exec jekyll serve
+
+# (optional) You only need to run `bundle exec` when you change dependencies
```

Comment:
I think this comment and the one on line 20 are swapped?

Reply:
Oh, I see now - you're saying that the command one line 21 is optional, only needs to be run when you change dependencies. It's a bit confusing as-is - if my understanding is correct, it would be better to have something like:
```
# Run the Jekyll development server (http://localhost:1924)
# If you changed the dependencies, you'll need to run `bundle exec`:
bundle exec jekyll serve
# Otherwise, you can just run:
jekyll serve
```

Reply:
When using RVM 1.29.9 with ruby 2.6.3p62 on Ubuntu 18.04, `jekyll serve` fails to run, producing the following error:
```bash
Command 'jekyll' not found, but can be installed with:

sudo apt install jekyll
```
Without installing all the gems specified in Gemfile to a location in $PATH (like $BUNDLE_PATH or $GEM_HOME), `jekyll serve` will not work. It might be beneficial to either include this information or remove the `jekyll serve` instruction.